src/models.js

```diff
@@ -1052,17 +1052,22 @@ module.exports = ({ cooler, isPublic }) => {
                 debug("getting root ancestor of %s", msg.key);
 
                 if (typeof msg.value.content === "string") {
-                  debug("private message");
                   // Private message we can't decrypt, stop looking for parents.
-                  resolve(parents);
-                }
-
-                if (msg.value.content.type !== "post") {
+                  debug("private message");
+                  if (parents.length > 0) {
+                    // If we already have some parents, return those.
+                    resolve(parents);
```

Comment:
Is it ever appropriate to resolve to an array? Everything seems to assume the resolved value is a msg.

Typescript plz.